WcaOnRails/db/migrate/20161026201019_simplify_registrations.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class SimplifyRegistrations < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+      CREATE TABLE old_registrations LIKE registrations;
+    SQL
+
+    # To keep things simple, just copy the entire registrations table over to
+    # old_registrations.
+    execute <<-SQL
+      INSERT INTO old_registrations SELECT registrations.* FROM registrations;
+    SQL
+
+    # Now delete all the registrations for competitions whose results have already
+    # been uploaded.
+    execute <<-SQL
+      DELETE registrations.*
+      FROM registrations
+      JOIN Competitions on Competitions.id=registrations.competitionId
+      WHERE Competitions.results_posted_at IS NOT NULL;
+    SQL
+
+    rename_column :registrations, :competitionId, :competition_id
+    remove_column :registrations, :name
+    remove_column :registrations, :personId
+    remove_column :registrations, :countryId
+    remove_column :registrations, :gender
+    remove_column :registrations, :birthYear
+    remove_column :registrations, :birthMonth
+    remove_column :registrations, :birthDay
+    remove_column :registrations, :email
+    remove_column :registrations, :guests_old
```

Comment:
You can use [`remove_columns`](http://apidock.com/rails/v4.2.7/ActiveRecord/ConnectionAdapters/SchemaStatements/remove_columns).